var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -32,6 +32,8 @@ class Jdk(Package):
     # found in a link above. The build number can be deciphered from the URL.
     # Alternatively, run `bin/java -version` after extracting. Replace '+'
     # symbol in version with '_', otherwise it will be interpreted as a variant
+    version('14', sha256='4639bbaecc9cc606f1a4b99fda1efcaefcbf57a7025b3828b095093a6c866afd',
```

Comment:
```suggestion
    version('14_36', sha256='4639bbaecc9cc606f1a4b99fda1efcaefcbf57a7025b3828b095093a6c866afd',
```
That's so weird. We can at least include the build version though.

Reply:
Fair enough. I'll get the pr updated, hopefully today.

Reply:
@anderbubble do you think you'll have a chance to update this in the next week or so?

Reply:
@scheibelp I'll try to get to it tomorrow evening. Sorry for the delay.

Reply:
@scheibelp I've updated the PR to include the build number in the version. Let me know if you have any other suggestions.